src/helpers/__tests__/documentation.js

```diff
@@ -0,0 +1,8 @@
+import { buildDocsContents } from "../documentation";
+import { input, output } from "./fixtures/documentation";
+
+describe("buildDocsContents", () => {
+  it("matches snapshot", () => {
+    expect(buildDocsContents(input.basic, "docs")).toStrictEqual(output.basic);
```

Comment:
Suuuper basic manual snapshot test. Basically just here so that if we have to touch this function again, can at least check happy path.

Reply:
I like this!

Reply:
this is great! i think something simple like this is a really good idea

Reply:
Cheers. My main complaint with snapshot testing is that it's super easy to become numbed to test failures. For asserting complex output doesn't change (like here) I think I like it.